card/src/main/java/com/adyen/checkout/card/CardComponentProvider.kt

```diff
@@ -54,15 +56,16 @@ class CardComponentProvider : StoredPaymentComponentProvider<CardComponent, Card
                 DefaultCardDelegate(
                     observerRepository = PaymentObserverRepository(),
                     publicKeyRepository = publicKeyRepository,
-                    configuration = verifiedConfiguration,
+                    configuration = configuration,
```

Comment:
Maybe a stupid question; but why do we need to pass both the configuration and params? Isn't only the params enough?

Reply:
We only need params, but I cannot remove the configuration yet because it's in the `ComponentDelegate` interface

Reply:
Ahh okay 👍